scripts/gen_profiles_solution.py

```diff
@@ -4207,6 +4207,9 @@ def applyWorkarounds(self):
         if self.headerVersionNumber.patch < 207: # vk.xml declares maxColorAttachments with 'bitmask' limittype before header 207
             self.structs['VkPhysicalDeviceLimits'].members['maxColorAttachments'].limittype = 'max'
 
+        if self.headerVersionNumber.patch < 304: # vk.xml declares minMemoryMapAlignment with 'min,pot' limittype before header 304 (and 'noauto' before 219)
```

Comment:
I think this should still be inside of `if 'VkPhysicalDeviceLimits' in self.structs:`

Reply:
Should the same fix be applied to the previous check in 4207? If that condition is not true, is there any chance of generating a useful profile anyway?

Reply:
Yes I believe so. If `VkPhysicalDeviceLimits` is not specified it is possible to just use device values